sensor_msgs/PointCloud2 pointCloud
ColorArray colorscam1
ColorArray colorscam2
